### PR TITLE
Use `composer serve` for tests, add tests for `windows`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: tests/Acceptance.suite.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ actor: AcceptanceTester
 extensions:
   enabled:
     - Codeception\Extension\RunProcess:
-        0: composer serve
+        0: yii serve --docroot=public --port=8881
```
Comment (samdark) [inline tests/Acceptance.suite.yml:5]: Why adjusting it?

Comment (terabytesoftw) [reply tests/Acceptance.suite.yml]: done.

**Comment by terabytesoftw** [comment]
merge first: https://github.com/yiisoft/actions/pull/73

**Comment by samdark** [comment]
@terabytesoftw done.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -22,11 +22,20 @@ on:
...
+  php-built-in-server:
     uses: yiisoft/actions/.github/workflows/codeception.yml@master
     with:
       extensions: fileinfo, intl
       os: >-
-        ['ubuntu-latest']
+        ['ubuntu-latest', 'windows-latest']
+      php: >-
+        ['8.1', '8.2', '8.3']
+  yii-serve:
```
Comment (samdark) [inline .github/workflows/build.yml:33]: Only this one should stay since we need to use our own tool instead of a workaround.

Comment (terabytesoftw) [reply .github/workflows/build.yml]: done.

**Comment by terabytesoftw** [comment]
@samdark done, tests only `yii serve`.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**